Sources/StreamChatUI/ChatMessageList/ChatMessageListView.swift

```diff
@@ -178,7 +178,7 @@ open class ChatMessageListView: UITableView, Customizable, ComponentsProvider {
         }
 
         switch change {
-        case let .insert(message, index: index):
+        case let .insert(_, index: index):
```

Comment:
😨 Did we actually release 4.15.0 with this warning?

Reply:
Yep, we did 😞 We need to add lint phase to Allure to prevent it from happening in the future